core/tests/test_utils.py

```diff
@@ -555,9 +555,30 @@ def delete_json(self, url, params='', expect_errors=False,
         return self._parse_json_response(
             json_response, expect_errors=expect_errors)
 
-    def _send_post_request(
+    def _send_post_request( # pylint: disable=missing-param-doc, missing-type-doc
             self, app, url, data, expect_errors=False, expected_status_int=200,
             upload_files=None, headers=None):
+        """Sends a post request with the data provided to the url specified.
+
+        Args:
```

Comment:
@seanlip I had to add the disable pylint statement because it was saying `missing-param-doc` for parameter "app" but it was clearly there. Also I noticed that changing `Args` to `Params` solved the issue but I didnt went ahead with it as it would break the style. PTAL!
Thanks.